### ISSUE TITLE
getting chromuim download error on v2.0.0

### ISSUE THREAD

**Issue by ranshaa05** [pyppeteer/pyppeteer#464]
hello, I'm trying to use the new version, but I seem to be getting this error:
`OSError: Chromium downloadable not found at https://storage.googleapis.com/chromium-browser-snapshots/Win_x64/1181205/chrome-win.zip: Received <?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: chromium-browser-snapshots/Win_x64/1181205/chrome-win.zip</Details></Error>.`

seems like the chromium zip for version 1181205 has been removed

**Comment by J4RF** [comment]
The package enforces a specific Chromium revision for no good reason, and in this release it's a revision that doesn't exist for Windows. Well done folks...

You can manually download the latest Chromium revision for Windows (.zip) here: https://chromium.woolyss.com/download/en/, then create a folder called 1181205 in C:\Users<username>\AppData\Local\pyppeteer\pyppeteer\local-chromium, and drag the extracted chrome-win folder into the 1181205 folder.

**Comment by ranshaa05** [comment]
> The package enforces a specific Chromium revision for no good reason, and in this release it's a revision that doesn't exist for Windows. Well done folks...
> 
> You can manually download the latest Chromium revision for Windows (.zip) here: https://chromium.woolyss.com/download/en/, then create a folder called 1181205 in C:\Users\AppData\Local\pyppeteer\pyppeteer\local-chromium, and drag the extracted chrome-win folder into the 1181205 folder.

Yup, already dove into the code and figured this out. unfortunately, the Google repository pyppeteer uses is quite convoluted and weird to work with...
I might have a look at a platform-specific download PR if I have the time.

**Comment by Mattwmaster58** [comment]
> The package enforces a specific Chromium revision for no good reason, and in this release it's a revision that doesn't exist for Windows. Well done folks...

Well, there was a good reason, and it was the same reason upstream puppeteer operates in the exact same way. By pinning a chrome version it reduces the testing burden and guarentees no problem with using unstable/experimental CDP APIs. 

Unfortunately I won't be updating this as this project is unmaintained
